### ISSUE TITLE
missing dotnet-tools.json in template content

### ISSUE THREAD

**Issue by jichang** [fable-compiler/fable-templates#36]
I just tried to create a fable project with this template according to website

```
dotnet new -i Fable.Template

dotnet new fable -n Demo

cd Demo

npm install

npm start
```
when I execute ```npm start```, it throws error

```
> SimpleFableApp@1.0.0 start /home/ms/Demo
> dotnet fable watch src --run webpack-dev-server

Could not execute because the specified command or file was not found.
Possible reasons for this include:
  * You misspelled a built-in dotnet command.
  * You intended to execute a .NET program, but dotnet-fable does not exist.
  * You intended to run a global tool, but a dotnet-prefixed executable with this name could not be found on the PATH.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! SimpleFableApp@1.0.0 start: `dotnet fable watch src --run webpack-dev-server`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the SimpleFableApp@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ms/.npm/_logs/2021-03-05T08_54_22_794Z-debug.log
```

seems it miss the .config/dotnet-tools.json, which exist in https://github.com/fable-compiler/fable3-samples/tree/main/minimal/.config

```
otnet --info
.NET SDK (reflecting any global.json):
 Version:   5.0.101
 Commit:    d05174dc5a

Runtime Environment:
 OS Name:     ubuntu
 OS Version:  20.04
 OS Platform: Linux
 RID:         ubuntu.20.04-x64
 Base Path:   /usr/share/dotnet/sdk/5.0.101/

Host (useful for support):
  Version: 5.0.1
  Commit:  b02e13abab

.NET SDKs installed:
  5.0.101 [/usr/share/dotnet/sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 5.0.1 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 5.0.1 [/usr/share/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET runtimes or SDKs:
  https://aka.ms/dotnet-download
```

Is this expected ? Or should we add that file ?

**Comment by MangelMaxime** [comment]
Hello @jichang, 

we should add the file in order to control the version of Fable from the project and not globally on the computer.

Would you like to send a PR?

**Comment by jichang** [comment]
@MangelMaxime sure, PR is here https://github.com/fable-compiler/fable-templates/pull/37

**Comment by MangelMaxime** [comment]
Thank you for the PR.

The changes have been released in 3.2.0
